ui/tests/unit/queries.spec.js

```diff
@@ -335,60 +335,56 @@ describe("Search", () => {
   };
 
   test.each([
```

Comment:
I think these tests should be in a different file. These are the tests for the queries.